content/working_with/official_plugins/Orchestration/helm.md

```diff
@@ -0,0 +1,465 @@
+---
+layout: bt_wiki
+title: Helm 3 Plugin
+category: Official Plugins
+draft: false
+weight: 100
+aliases: ["/plugins/helm/", "/developer/official_plugins/helm/"]
+---
+
+Helm is the first application package manager running on top of Kubernetes. It allows describing the application structure through convenient helm-charts and managing it with simple commands.
+Helm uses three main concepts:
+
+* Charts
+
+* Repository
+
+* Release
+
+A Chart is a Helm package. It contains all of the resource definitions necessary to run an application, tool, or service inside of a Kubernetes cluster. Think of it like the Kubernetes equivalent of a Homebrew formula, an Apt pkg, or a Yum RPM file.
+
+A Repository is a place where charts can be collected and shared. It's like Perl's CPAN archive or the Fedora package Database, but for Kubernetes packages.
+
+A Release is an instance of a chart running in a Kubernetes cluster. One chart can often be installed many times into the same cluster. And each time it is installed, a new release is created. Consider a MySQL chart. If you want two databases running in your cluster, you can install that chart twice. Each one will have its own release, which will in turn have its own release name.
+
+With cloudify Helm 3 plugin you can add repositories and create releases on Kubernetes cluster.
+
+
+# Plugin Requirements
+
+* Python versions:
+  * 2.7.x/3.6.x
+* Kubernetes Cluster, see [example cluster](https://github.com/cloudify-community/blueprint-examples/tree/master/kubernetes).
+
+In order to know which versions of Kubernetes Helm supports,see [Helm version support policy](https://helm.sh/docs/topics/version_skew/)
+
+
+## Authentication
+In order helm can interact with Kubernetes cluster, Authentication is needed.
+There is only single authentication method which is kube config authentication, the authentication is pretty similar as in cloudify kubernetes plugin.
+
+### Kube Config Authentication
+
+Authentication with the Kubernetes cluster is done under "cclent_config.configuration" section in "cloudify.nodes.helm.Release" node type. The config should be a [Kube Config style](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#define-clusters-users-and-contexts) object.
```

Comment:
Rewrite.
To configure authentication with Kubernetes, use blah blah blah.

Reply:
Done